src/components/Transfer/Source.tsx

```diff
@@ -260,6 +267,8 @@ function Source() {
               value={amount}
               onChange={handleAmountChange}
               disabled={isTransferDisabled || shouldLockFields}
+              error={isBelowMinimum}
+              helperText={isBelowMinimum ? "Amount is below minimum" : ""}
```

Comment:
Maybe the error message could be more helpful? “Amount sent is too small. The amount must be equal or greater than X.” Or something like that

Reply:
sure

Reply:
pushed an updated version with this change

Reply:
<img width="925" alt="image" src="https://github.com/XLabs/portal-bridge-ui/assets/1277510/bd49b1d3-376f-4dac-969e-dfb37d5d8a04">